Makefile

```diff
@@ -64,6 +64,21 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Use podman if it's available
```

Comment:
@dtadcox @pbaity could you please review these proposed changes to the _Makefile_ and confirm that they will not affect the pipeline? Podman will be used if it's available.

OnePipeline doesn't call `catalog-build`, but it calls others I believe.

Reply:
1) This copy of the Makefile is stale, so merging may be difficult.
2) The bundle-push rule was altered in a way that I don't believe it should have been.  That rule deferred to the docker-push rule for implementation, but now it has been changed to work independently.  I don't see a need for that change.  It could be argued that it isn't needed at all, but for consistency sake, I would keep it working as is.  The change to facilitate the container tool switching is handled by the docker-push rule, so just allow it to work as is.
3) this is a general comment, but for pipeline catalog build, a new set of rules were added to handle that activity.  It is confusing in the makefile, that there are two other catalog-build rules.  'catalog-build', 'catalog-push', 'build-catalog' and 'push-catalog'.  And now, we have 'catalog-pipeline-build' and 'catalog-pipeline-push'.  So, we need to keep the pipeline ones as they are very different in implementation than the others, but lets settle on either 'catalog-build' or 'build-catalog' and remove the other two.  
4) Last concern, not sure it is valid, is that the change to docker-push rule will now pick up whichever tool is found.  I believe podman has first crack at it, then docker.  So, in the current implementation of 'catalog-pipeline-push', we are simply deferring to the action in 'docker-push' to perform the push.  This should be fine with this update, but if somehow a version of podman shows up in the one-pipeline VMs, then we might have an issue.  As stated, this is probably not a concern, but i wanted to raise it.  I'm thinking, do we want some type of 'override' ability in the makefile?  Where i can set aa flag to force a container tool to be used?  Maybe thats there already and i missed it?  Would be nice, i think, to set an environment variable, say in our 'catalog-pipeline-push' rule that forces the use of docker.

Reply:
To answer my point #4 - should we simply override the value of 'CONTAINER_COMMAND' prior to performing any execution in the *pipeline* rules?

Reply:
@dtadcox 
Item 1 - 3,  Are merge related concerns.  
Item 4,  Handled by favoring docker first when available.  It can be overridden by setting CONTAINER_COMMAND